gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ activemq-artemis = "2.33.0"
 application-insights = "2.6.4"
 archunit = "1.3.0"
 asmForPlugins = "7.3.1"
-aspectjweaver = "1.9.22.1"
+aspectjweaver = "1.9.20.1"
```

Comment:
1.9.20.1 is the last release that accepts jdk 11 for building https://github.com/eclipse-aspectj/aspectj/blob/master/docs/release/JavaVersionCompatibility.adoc

Reply:
If we want to avoid bumping this version in the future, we should leave a comment here so we know why. I'll add it as a polish commit.